docs/versioned_docs/version-v1.5.0/features/hosting/formatting.md

```diff
@@ -31,12 +43,31 @@ public class Startup
     }
 }
 ```
-
+</details>
```

Comment:
```suggestion

</details>
```
There needs to be some blank line before and after the `<details>` tags otherwise Docusaurus wouldn't like it.
See: https://docusaurus.io/docs/next/markdown-features/react#markdown-and-jsx-interoperability